sql/core/src/test/resources/sql-tests/analyzer-results/grouping_set.sql.out

```diff
@@ -159,28 +159,42 @@ Sort [-col1#x ASC NULLS FIRST], true
 -- !query
 SELECT a, b, c, count(d) FROM grouping GROUP BY WITH ROLLUP
```

Comment:
This case is fine as we just changed the error message. Is there a case that we actually change semantic as now we allow column alias in the GROUP BY clause?

Reply:
I tried and I can't think of any. The new optional `AS <alias>` or `<alias>` part at the end of each grouping key would result in syntax errors before.